homeassistant/components/camera/bomradarcam.py

```diff
@@ -0,0 +1,75 @@
+"""
+Provide animated GIF loops of BOM radar imagery.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/bomradarcam/
```

Comment:
We don't include the url in the docstring anymore. Just keep the very first line.